packages/models-library/src/models_library/generics.py

```diff
@@ -51,6 +52,11 @@ def setdefault(self, key: DictKey, default: DictValue):
     def __len__(self) -> int:
         return self.__root__.__len__()
 
+    def dict(self, *args, **kwargs) -> Dict[str, Any]:
+        # `this is due to the fact that FastAPI does not appear to like pydantic models with just a root`
+        data = super().dict(*args, **kwargs)
```

Comment:
IMO this is pretty dangerous ... and I would test this change very carefully. See for instance the implementation of members as [``parse_obj``](https://github.com/samuelcolvin/pydantic/blob/master/pydantic/main.py#L506)

As I suggest below, I still wonder what is the real advantage of ``DictModel[DictKey, DictValue]`` versus simply ``dict[DictKey, DictValue]``. The latter is also parsable via pydantic and FastAPI will not complain!

I highly recommend NOT doing this change and rather a) use ``dict[DictKey, DictValue]`` or b) convert your ``DictModel`` to a ``dict`` in place e.g. in fastapi return you do ``return dict(mydictmodel.__root__)``

Reply:
As soon as the CI goes green, I will drop the `DictModel` and try to use `dict` as suggested.